fmperf/loadgen/collect_energy.py

```diff
@@ -123,19 +124,19 @@ def get_file_prefix(start_ts: str):
 
 # read metrics files and concatenate them to integrate performance data
 def summarize_energy(start_ts: str):
+    metric_list = os.environ.get('TARGET_METRICS_LIST', 'default_metrics.yaml') 
     all_df = pd.DataFrame(dtype=float)
     # target metrics
-    metrics = [
-        "DCGM_FI_DEV_POWER_USAGE",
-        "DCGM_FI_DEV_GPU_UTIL",
-        "DCGM_FI_DEV_MEM_COPY_UTIL",
-        "DCGM_FI_PROF_PIPE_TENSOR_ACTIVE",
-        "DCGM_FI_PROF_SM_ACTIVE",
-        "DCGM_FI_PROF_SM_OCCUPANCY",
+    metrics = ["DCGM_FI_DEV_POWER_USAGE",
         "kepler_container_gpu_joules_total",
         "kepler_container_package_joules_total",
-        "kepler_container_dram_joules_total",
-    ]
+        "kepler_container_dram_joules_total"]
+    if metric_list is not None:
+         with open(metric_list, 'r') as yml:
+             config = yaml.safe_load(yml)
+             mlist = config["metrics"]
+             metrics.extend(mlist)    
```

Comment:
directly extending `metrics` column name by `mlist` can cause redundant column name. The default `metrics` already contain `kepler_container_gpu_joules_total`, `kepler_container_package_joules_total`, and `kepler_container_dram_joules_total`. However, in your `fmperf/loadgen/default_metrics.yaml`, these three are also included. Consider checking if the column name wanted to add is unique before adding a new column name